infra/gcp/terraform/README.md

```diff
@@ -55,9 +56,49 @@ containing manifests that are deployed to the cluster
 
 ## Deleting
 
-- Get approval from a WG K8s Infra lead (ask in [#sig-k8s-infra] before doing this)
+- Get approval from a SIG K8s Infra lead (ask in [#sig-k8s-infra] before doing this)
 - Ensure you are logged into your GCP account with `gcloud auth application-default login`
 - From within a module directory:
   - `terraform destroy` will destroy and clean up all created resources
 
 [#sig-k8s-infra]: https://kubernetes.slack.com/messages/sig-k8s-infra
+
+
+# Bootstrapping Terraform - One Time Setup
+
+Terraform needs to be bootstrapped manually before it can be used. This process was done during Atlantis Setup. It is noted here for completeness and for potential troubleshooting.
+
+This needs to be ran by a person.
+
+```
+# Get the ORG_ID
+ORG_ID=$(gcloud organizations describe kubernetes.io --format json | jq .name -r | sed 's:.*/::')
```

Comment:
Or:
```suggestion
ORG_ID=$(gcloud organizations describe kubernetes.io --format="value(name)" | sed 's:.*/::')
```
To avoid jq installation ?

Reply:
I'll run these commands right now and the won't be run ever again.